postgresql-dedicated-clusters-beta.mdx

```diff
@@ -0,0 +1,132 @@
+
+We're extremely excited to announce the next milestone in this evolution, a public beta for dedicated clusters 🎉
+
+We also want to whole-heartedly thank our EAP customers for not only being early adopters, but providing amazing levels of feedback throughout the journey. Keep a look out, you'll all be receiving some premium Xata swag soon 💜
+
+## What is a dedicated cluster?
+
+Today, most Xata databases live in a shared cell architecture. Our multi-tenant cells include a Postgres Aurora cluster, an Elasticsearch cluster, high availability built in and all the APIs and services required for development. Within each cell, we split the system for each workspace to include about 2,000 active databases per cell before we have to spin up a new one. Unless you were an EAP customer, your Xata database is using this shared cell architecture.
+
+Our dedicated clusters are essentially dedicated cells. So you have all the same benefits of the shared cell architecture, with less limitations caused by shared resources, more control over your infrastructure, decoupled usage based pricing and the confidence that Xata can scale with your business.
+
+![Postgres branches allocated to shared and dedicated clusters](images/branches-clusters.png)
+
+I won't get into the weeds today, but if you're interested in learning more, [this blog](https://xata.io/blog/postgres-dedicated-clusters) and [our documentation](https://xata.io/docs/dedicated-cluster) are great resources.
+
+## Celebrating with a limited-time offer
+
+To kick off this [launch week](https://xata.io/launch-week-elephant-on-the-move) right and celebrate our public beta release, we will be offering **$1,000 in credit** for customers that start using dedicated cluster between now and its general availability later this year. This credit will be valid through the end of 2025, but you will only have a few months to qualify. As a customer, you do not need to apply for the credit. Any usage from a dedicated cluster will automatically be deducted and you have full control over what the deployment looks like.
+
+If you're already a happy Xata customer, you can easily move all of your databases or a single branch to a dedicated cluster in just a few clicks.
+
+## What's new
+
+The team has been hard at work over the last few months building out features and functionality to provide the best experience to our beta customers. Starting today you can spin up dedicated Postgres cluster classes ranging in size from `db.t4g.medium` to `db.r6g.2xlarge` in both `us-east-1` and `eu-central-1`. We plan to introduce more classes, versions, regions and extensions in the coming months, so [please let us know](https://xata.io/discord) what you'd like to see.
```

Comment:
```suggestion
The team has been hard at work over the last few months building out features and functionality to provide the best experience to our beta customers. Starting today you can spin up dedicated Postgres cluster classes ranging in size from `db.t4g.medium` to `db.r6g.2xlarge` in both `us-east-1` and `eu-central-1`. We plan to introduce more classes, versions, regions and extensions in the coming months, so [please let us know](https://xata.io/contact) what you'd like to see.
```

Reply:
might be a good idea to link direct to contact? since discord filters for people who have/use discord

Reply:
I like it